src/osp/Universe.h

```diff
@@ -135,6 +135,14 @@ class Universe
         return m_satTypes.find(name);
     }
 
+    /**
+     * Tries to locate an element in the map of registered Satellite by type.
+     * @tparam SATTYPE_T Type of ITypeSatellite to find
+     * @return Map iterator directly from std::map::find()
```

Comment:
Is it returning an iterator? or a reference?

If returning an iterator, i don't think it makes sense to templatize it?